src/data/code-samples/en/text-bold/net-maui.md

```diff
@@ -1,5 +1,29 @@
 # Bold text - .NET MAUI
 
+While MAUI doesn't offer a way to detect if the user has enabled bold fonts, you can create a method and consume the Native API to detect user preferences.
```

Comment:
Maybe rewrite for consistency:
`In MAUI, there is no way to detect if the user has enabled bold font, but you can create a method and consume the Native API to detect user preferences.`